.gitignore

```diff
@@ -21,3 +21,6 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+# typechain
+types/ethers-contracts
```

Comment:
dumb question - should this folder contain the types generated in `saddle-contracts` repo?

Reply:
It's a subset of the generated types as the `saddle-contract` repo, since we only copied some ABIs into `src/constants/abis/` here.